javascript/ql/src/Security/CWE-094/ImproperCodeSanitization.qhelp

```diff
@@ -3,35 +3,36 @@
   "qhelp.dtd">
 <qhelp>
 
-<overview>
-<p>
-Placeholder
-</p>
-</overview>
+  <overview>
+    <p>
+        Using string concatenation to construct JavaScript code can be error-prone, or in the worst
+        case enable code-injection if an input is constructed by an attacker.  
+    </p>
+  </overview>
 
-<recommendation>
-<p>
-Placeholder
-</p>
-</recommendation>
+  <recommendation>
+    <p>
+      If using <code>JSON.stringify</code> or a HTML sanitizer to sanitize a string inserted into 
+      JavaScript code, then make sure to perform additional sanitization or remove potentially
+      dangerous characters. 
+    </p>
+  </recommendation>
 
-<example>
-<p>
-Placeholder
-</p>
+  <example>
+    <p>
+      The below example constructs a function that assigns the number 42 to the property <code>key</code>
+      on an object <code>obj</code>. However, if <code>key</code> contains <code>&lt;/script&gt;</code>, then 
+      the generated code will break out of a <code>&lt;script&gt;</code> if the generated code is inserted 
```

Comment:
This sentence is a bit repetitive. What about:
... then the generated code will break out of a <script> if inserted into a <script> tag.